catalyst_voices/lib/widgets/rich_text/voices_rich_text.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_quill/flutter_quill.dart';
+
+/// A component for rich text writing
+/// using Quill under the hood
+/// https://pub.dev/packages/flutter_quill
+class VoicesRichText extends StatefulWidget {
```

Comment:
We may need to have a few pre-configured options. How can we customize the text editor? 
How do we access data in this widget (QuillController) from another widget or pass data to this widget (QuillController)?

Reply:
I wasn't going heavy with this example before Niels gives a green light that it suits him from UX/UI perspective. I understand from our meetings that they prefer fast iterations and things to play quickly instead of polishing before shipment, but if you find mandatory to do it right now, I can add it.

Regarding passing data: yes `QuillController`.
I also checked that we can add more advance things like commenting with custom attribute implementation https://github.com/singerdmx/flutter-quill/issues/1869 I also left it for the separate PR, but I can also play with it here If you prefer.

Reply:
Let's add the possibility of passing data both ways, please.

Reply:
Alright, currently I'm passing `Document` and callback `ValueChanged<Document>? onSave`